### PR TITLE
style tweaks, resize player

### PR DESCRIPTION
**Pull request by tyard**

### CODE REVIEW

File: css/bcgs_ford_main.css [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
-	src: url('fonts/regular/extralight.eot?#iefix') format('embedded-opentype'),
-	     url('fonts/regular/extralight.woff') format('woff'),
-	     url('fonts/regular/extralight.ttf') format('truetype'),
-	     url('fonts/regular/extralight.svg#webfont') format('svg');
+    font-family: 'antennaExtraLight';
+    src: url('fonts/regular/extralight.eot');
+    src: url('fonts/regular/extralight.eot?#iefix') format('embedded-opentype'),
+         url('fonts/regular/extralight.woff') format('woff'),
+         url('fonts/regular/extralight.ttf') format('truetype'),
+         url('fonts/regular/extralight.svg#webfont') format('svg');
```
Comment (tyard) [inline css/bcgs_ford_main.css]: Just indentation change -- replaced tabs.

File: css/fonts/regular/_notes/dwsync.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (tyard) [inline css/fonts/regular/_notes/dwsync.xml]: hmm, added this to ignore -- not sure why it got pushed.

**Comment by bceacuna** [comment]
Looks good.

File: lib/bcgs_ford_modal.js [hunk 3]
```diff
@@ -98,6 +103,31 @@ TODO: remove instrinsic ratio implementation from index.html and it to the plugi
...
+        sizeModal = function() {
+            var aspectRatio = 0.5624, /* 703/1250 */
+                windowHeight = $(window).height(),
+                maxWidth = $(window).width()-margin*2,
+                maxHeight = windowHeight-margin*2,
+                minHeight = maxWidth*aspectRatio+openDrawerHeight,
+                modalWidth = minHeight > maxHeight ? (maxHeight-openDrawerHeight)/aspectRatio : maxWidth,
+                videoHeight = modalWidth*aspectRatio,
+                modalHeight = videoHeight;
+
```
Comment (bceacuna) [inline lib/bcgs_ford_modal.js]: I would suggest expressing the aspectRatio as a fraction (16/9).

File: lib/bcgs_ford_modal.js [hunk 4]
```diff
@@ -43,15 +43,18 @@ TODO: remove instrinsic ratio implementation from index.html and it to the plugi
     log(NAME + " loaded.");
     // TODO: rename css class names (something more descriptives)
     var video,
+        $modal = $("#modal-container"),
+        margin = 50, /* @todo: get this from CSS */
+        drawerHeight = 100,
+        openDrawerHeight = 250,
         $closeButtonDiv = $('<div class="bcc-modal-close-button">x</div>'),
```
Comment (bceacuna) [inline lib/bcgs_ford_modal.js]: The close button icon has been replaced with an image. So the character "x" should be removed

File: lib/bcgs_ford_modal.js (hunk 3 above)
Comment (tyard) [inline lib/bcgs_ford_modal.js]: I was taking from the CSS that was there, which 1250/703, which is not quite 16/9. We can certainly use that (it's cleaner), but I didn't know if that matched exactly the aspect ratio you had discovered and put in the CSS. I'm for updating it, though.

File: lib/bcgs_ford_modal.js (hunk 4 above)
Comment (tyard) [inline lib/bcgs_ford_modal.js]: Yep, missed that.

File: css/bcgs_ford_main.css [hunk 5]
```diff
@@ -41,32 +39,25 @@
...
-    height: 1.2em; 
+    width: 66px; 
+    height: 65px; 
     top: 0px;
     right: 0px;
-    opacity: 0.85;
-    background: #121111;
-    color: white;
+    background-image: url(../img/close-icon.png);
 }
```
Comment (bceacuna) [inline css/bcgs_ford_main.css]: Shouldn't be using an svg instead, so the image is properly rendered/resized on devices with a high PPI.

Comment (tyard) [inline css/bcgs_ford_main.css]: Perhaps, but I've never seen much issue with it, and at this time I was just rushing to get something out to Alfredo that looked like the designs. We can revisit as we test on different sizes. What was there before wasn't SVG so this change wasn't to remove it anyway.

**Pull request merged**